eng/test-determinism.ps1

```diff
@@ -24,8 +24,12 @@ if ($help) {
 # makes them non-deterministic.
 $script:skipList = @(
   # Added to work around https://github.com/dotnet/roslyn/issues/48417
-  "Microsoft.CodeAnalysis.EditorFeatures2.UnitTests.dll"
+  "Microsoft.CodeAnalysis.EditorFeatures2.UnitTests.dll",
+
+  # TOOD: file XLF additional files bug before merge
```

Comment:
nit: FWIW, I've added "TODO2" to the list of words checked by the CI leg. Unlike "PROTOTYPE" it is checked in all branches. It's handy to track this sort of thing.

Reply:
Got the issue filed for this.